assets/css/style.scss

```diff
@@ -185,8 +187,8 @@ $spacing-unit:     30px !default;
   position: absolute;
   bottom: 10px;
   left: 14px;
-  color: white;
-  background: #2b2727b0;
+  color: $text-white;
+  background: $bg-gray-dark;
```

Comment:
I added both `banner` and `banner-credit` classes to style (they do not belong to original template style), so they don't necessarily need to inherit template style; particularly regarding the `background` change, I wanted to find a level of opacity that somehow matches the title one, while still keeping the photo credits readable.

##### Without your proposed change

![image](https://github.com/user-attachments/assets/98735fef-06f1-4e02-9802-fd4aa1ba142f)

##### With your proposed change

![image](https://github.com/user-attachments/assets/bf78c6cb-3401-4d92-aea8-6d786ca00636)

Reply:
Right, I didn't spot the opacity, go ahead and revert that.

Reply:
Reverted in 4552c5b4388bec73d3bcec1dfdc8db56e823ec77.